src/main/kotlin/no/nav/rekrutteringsbistand/stillingssokproxy/Main.kt

```diff
@@ -8,6 +11,30 @@ val Any.log: Logger
 
 fun log(name: String): Logger = LoggerFactory.getLogger(name)
 
+val environment = dotenv { ignoreIfMissing = true }
+
 fun main() {
     log("main").info("Starter applikasjon")
+
+    val port = environment["JAVALIN_PORT"].toInt()
+    val aliveUrl = "http://localhost:${port}/internal/isAlive"
+    val readyUrl = "http://localhost:${port}/internal/isReady"
+
+    val javalin = Javalin.create().start(port)
+
+    if (environment["NAIS_CLUSTER_NAME"] == "local") {
+        log("main").warn("Applikasjonen settes opp med konfigurasjon for lokal kjøring")
+        javalin.before { context ->
+            context.cookieStore("innloggetVeileder", Security.InnloggetVeileder("brukernavn", "visningsnavn", "navident"))
+        }
+    } else {
+        val security = Security()
+        security.lagSikkerhetsfilter(javalin, listOf(aliveUrl, readyUrl))
+    }
+
+    javalin.routes {
+        get(aliveUrl) { it.status(200) }
+        get(readyUrl) { it.status(200) }
+        get("/test") { it.status(200) }
```

Comment:
Kunne man laget en test for dette? At man får 403 på lukka endepunkt og 200 på de åpne

Reply:
Vi har nå laget tester for dette.